rules/feature_select.smk

```diff
@@ -12,7 +12,7 @@ rule feature_select:
     params:
         feature_select_config=config["config_paths"]["feature_select"],
     log:
-        "logs/feature_select.log"
+        "logs/feature_select.log",
```

Comment:
I think black changed this. It doesn't look right to me!

Reply:
`black` is not used in `.smk` files it is actually `snakefmt` formatter. I do not know why that comma was not added. weird... Thanks for pointing that out.